src/Umbraco.Web/Models/Mapping/UserMapDefinition.cs

```diff
@@ -167,6 +167,7 @@ private void Map(IUserGroup source, UserGroupBasic target, MapperContext context
         // Umbraco.Code.MapAll -Udi -Trashed -AdditionalData -Id -AssignedPermissions
```

Comment:
Note the `-Id` here: the `// Umbraco.Code.MapAll` comment triggers a Roslyn Analyzer (that is included via NuGet) that verifies that all `target` fields are assigned a value - except those that are excluded with a `-Value` comment. So... `Id` was excluded, quite probably because AutoMapper *did* exclude it too. How did it get mapped, then? No idea.

Going to merge the PR - and *then* to remove the `-Id` in the comment, if we *do* want it to be mapped.